waiter/src/waiter/handler.clj

```diff
@@ -42,22 +42,29 @@
   {:username (:authorization/user request)
    :principal (:authenticated-principal request)})
 
+(defn async-make-request-helper
```

Comment:
As it is a helper function for `async-make-http-request`. Happy to rename, any suggestions?